src/Analyzers/Core/CodeFixes/GenerateConstructor/AbstractGenerateConstructorService.State.cs

```diff
@@ -574,7 +585,12 @@ private IEnumerable<string> GetUnavailableMemberNames()
 
             Contract.ThrowIfNull(TypeToGenerateIn);
 
+#if CODE_STYLE
+            var provider = document.Project.Solution.Workspace.Services.GetExtendedLanguageServices(TypeToGenerateIn.Language);
+#else
             var provider = document.Project.Solution.Services.GetLanguageServices(TypeToGenerateIn.Language);
```

Comment:
@tmat would love a way to abstract this behind something (but these helpers have genuinely different types).   i'll look into this myself.  But maybe you have some good ideas here.  This pattern shows up in a lot of files.